packages/app/package.json

```diff
@@ -61,7 +61,6 @@
   },
   "devDependencies": {
     "@h5web/shared": "workspace:*",
-    "@juggle/resize-observer": "3.3.1",
```

Comment:
Replacing this with a dumb `ResizeObserver` mock, which works just as well and saves a lot of computations since the polyfill is quite complex.